db/seeds.rb

```diff
@@ -323,3 +323,12 @@ def create_repository(link:, company:)
   repositories: repos,
   dates: contributions_dates
 )
+
+print 'Creating mentors to some users..'
+
+30.times do
+  mentor = User.all.sample
+  User.all.sample.update(reviewer_id: mentor.id)
```

Comment:
I know that performance on the seed file is not that big of a deal, but I suggest we pass this `User.all` to a variable before using it on lines 330 and 331 so we don't end up pulling all the Users two times.